Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -335,11 +335,17 @@ def audit_formula_name
       openssl@1.1
     ].freeze
 
+    PERMITTED_LICENSE_MISMATCHES = [
+      %w[GPL-3.0-only GPL-3.0-or-later GPL-3.0],
+      %w[GPL-2.0-only GPL-2.0-or-later GPL-2.0]
+    ]
+
     def audit_license
       if formula.license.present?
         non_standard_licenses = formula.license.map do |license|
           next if license == :public_domain
           next if @spdx_data["licenses"].any? { |spdx| spdx["licenseId"] == license }
+          next if license == "NOASSERTION" && !@core_tap
```

Comment:
Does it make sense to have that as a `license` field entry? What makes it more useful than just `license`?

Reply:
This is handled by `363` below where you can specify whatever you want as the value if `NOASSERTION` is returned. I don't see the value in having that as an actual `license` value in a formula?

Reply:
Ahh, good point, yeah I agree.